.storybook/stories/KivaMultiCategoryGrid.stories.js

```diff
@@ -0,0 +1,168 @@
+import KivaClassicLoanCarousel from '@/components/LoanCollections/KivaClassicLoanCarousel';
+import apolloStoryMixin from '../mixins/apollo-story-mixin';
+import cookieStoreStoryMixin from '../mixins/cookie-store-story-mixin';
+import LoanCategorySelectorHomeExp from '@/components/LoanCollections/HomeExp/LoanCategorySelectorHomeExp';
+
+export default {
+	title: 'New Home Page/Kiva Multi Category Grid',
+	components: {LoanCategorySelectorHomeExp, KivaClassicLoanCarousel},
+	args: {
+		showAllButton: true,
+		combinedLoanChannelData: [
```

Comment:
You could consider moving these large data objects into the fixtures folder. See other stories/tests for examples.